Tensile/KernelWriterAssembly.py

```diff
@@ -41,11 +41,16 @@ def __init__(self, name, numAddresses, numOffsets, \
     self.endLine = "\n"
   ########################################
   # write in assembly format
-  def toString(self, params, comment, nonTemporal=0, highBits=0):
+  def toString(self, params, comment, nonTemporal=0, highBits=0, lds=0):
     name = self.name
     if highBits:
       name += "_d16_hi"
-    instStr = "%s %s" % (name, (self.formatting % params) )
+    # Remove destination from format string
+    # TODO-LDSDestBug - don't set lds flag to 0 here:
+    lds = 0
+    f = self.formatting.split(",",1)[-1] if lds else self.formatting
```

Comment:
Grr cannot select more than one line. If lds is set to 0, how does this line work?

Reply:
thx, good catch.   LLVM matches SP3 assembler and requires a dummy destination VGPR even for direct-to-lds.  I will remove this lds=0 in the next checkin, i think we just live with the asm syntax we have.